src/Microsoft.AspNet.Security.OAuth/OAuthBearerAuthenticationMiddleware.cs

```diff
@@ -2,21 +2,25 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using Microsoft.AspNet.Builder;
-using Microsoft.AspNet.Security.DataHandler;
 using Microsoft.AspNet.Security.DataProtection;
 using Microsoft.AspNet.Security.Infrastructure;
 using Microsoft.Framework.Logging;
 using Microsoft.Framework.OptionsModel;
+using Microsoft.IdentityModel.Protocols;
 using System;
+using System.Collections.ObjectModel;
+using System.Diagnostics.CodeAnalysis;
+using System.IdentityModel.Tokens;
+using System.Net.Http;
 
 namespace Microsoft.AspNet.Security.OAuth
 {
-    /// <summary>
-    /// Bearer authentication middleware component which is added to an HTTP pipeline. This class is not
-    /// created by application code directly, instead it is added by calling the the IAppBuilder UseOAuthBearerAuthentication
-    /// extension method.
-    /// </summary>
-    public class OAuthBearerAuthenticationMiddleware : AuthenticationMiddleware<OAuthBearerAuthenticationOptions>
+	/// <summary>
+	/// Bearer authentication middleware component which is added to an HTTP pipeline. This class is not
+	/// created by application code directly, instead it is added by calling the the IAppBuilder UseOAuthBearerAuthentication
+	/// extension method.
+	/// </summary>
+	public class OAuthBearerAuthenticationMiddleware : AuthenticationMiddleware<OAuthBearerAuthenticationOptions>
```

Comment:
Removing the `AccessTokenFormat`/`TicketDataFormat` stuff also means that people wanting to use opaque tokens won't be able to use the new OAuth2 bearer middleware without writing a custom `SecurityTokenHandler`. Looks like a too much aggressive change.